guessthenumber/number.js

```diff
@@ -0,0 +1,36 @@
+alert("Guess a number between 1 and 100!")
+
+var random = Math.floor((Math.random() * 100) + 1);
+var counter = 0;
+var maxGuesses = 5;
+var guess = 0;
+
+document.getElementById("checkButton").onclick = function() {
+    var guess = document.getElementById("guess").value;
+    checkGuess(guess);
+    counter++;
+
+    if (counter == maxGuesses) {
+        alert("Game Over");
+        document.getElementById("checkButton").disabled = true;
+    }
+}
+
+function checkGuess(guess) {
+    if (guess == random) { 
+        alert("You win!");
+        document.getElementById("checkButton").disabled = true;
+    }
+    if (guess > random) {
+        alert("Too high!");
+        document.getElementById("guess").value = '';
+    } else if (guess < random) {
+        alert("Too low!");
+        document.getElementById("guess").value = '';
+    }
```

Comment:
You could refactor the second and third else if in to one ternary e.g. 
```
else { 
  guess > random ? (lines 25-26) : (lines 28-29)
}
```
or make the second if an else if and the third would not need an else if declaration because all other possible instances have been covered by the previous conditions.